packages/@aws-cdk/aws-apigatewayv2-integrations/README.md

```diff
@@ -149,6 +150,56 @@ const httpEndpoint = new HttpApi(stack, 'HttpProxyPrivateApi', {
 });
 ```
 
+### Request parameter mapping
+
+### Request Parameters
+
+Request parameter mapping is supported through a set of helper methods.
+The following example renames a request header from header1 to header2 by generating mappings
+`append:header.header2: $request.header.header1` and `remove:header.header1: ''`:
+
+```ts
+const vpc = new ec2.Vpc(stack, 'VPC');
+const lb = new elbv2.ALB(stack, 'lb', { vpc });
+const listener = lb.addListener('listener', { port: 80 });
+listener.addTargets('target', {
+  port: 80,
+});
+
+const httpEndpoint = new HttpApi(stack, 'HttpProxyPrivateApi', {
+  defaultIntegration: new HttpAlbIntegration({
+    listener,
+    requestParameters: new RequestParameters()
+      .addParameter({
```

Comment:
How about calling this `ParameterMapping` instead? Because it's really defining a mapping for request parameters.
It can be re-used in the future for `responseParameters`.

Do we need the extra `addParameter()` here? Would it be simpler to just be something like -

```suggestion
    requestParameters: new ParameterMapping()
      .appendHeader('header2', MappingValue.header('header1'))
      .removeHeader('header1');
```

Reply:
Agree on the name change, I thought about that too. 

The reason I used `addParameter()` is that the [design guidelines](https://github.com/aws/aws-cdk/blob/master/docs/DESIGN_GUIDELINES.md) specifically mentioned that we should "prefer additions" and has lots of examples with an `add`-prefix. But I'm all for dropping it to simplify :-)

Reply:
I'm ok with not using the `add` prefix in this case.